libphal/exception.H

```diff
@@ -84,7 +90,7 @@ class SbeError final : public LibphalError
 
 } // namespace exception
 
-using libphalerror_t = exception::LibphalError;
-using sbeerror_t = exception::SbeError;
+using libphalError_t = exception::LibphalError;
+using sbeError_t = exception::SbeError;
```

Comment:
Please refactor your patches to include such changes when you introduce those terms.

Reply:
looks like some rebase mess, i will fix